examples/run/sycl/seq_example_sycl.sycl

```diff
@@ -104,6 +104,7 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
 
     // Struct with memory resources to pass to SYCL algorithms
     traccc::memory_resource mr{device_mr, &sycl_host_mr};
+    // traccc::memory_resource mr{shared_mr};
```

Comment:
Let's remove this line from this PR. We can come back to a nice setup for switching between memory resource setups in a future PR. :wink:

Reply:
:confused: You resolved this discussion, but the commented line is still there...